docs/secure-connections/sauce-connect/advanced/sc-kgp.md

```diff
@@ -0,0 +1,46 @@
+---
+id: sc-kgp
+title: KGP
+sidebar_label: Tunneling Protocol
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
```

Comment:
In our docs, we just go straight into the overview paragraph (no need for the ## Overview header)